src/plugin.ts

```diff
@@ -696,7 +692,12 @@ class Plugin {
     // add ClickLayer
     this.clickLayer.className = 'ima-click-layer'
     videoElement.after(this.clickLayer)
-    this.clickLayer.addEventListener('click', this.onVideoElementPlay)
+    this.clickLayer.addEventListener('click', (event: Event) => {
+      // user interaction triggers also the content video for playing later
+      videoElement.play()
+      videoElement.pause()
+      this.onVideoElementPlay(event)
```

Comment:
Inside onVideoElementPlay is `videoElement.pause()` too. I this also needed now?

Reply:
Not necessarily because it is called in the onVideoElementPlay. But I thought it would minimize the possibility to see the content video play and then the ad. What do you think?

Reply:
If you think this is better for the user it's fine for me (I don't know the whole structure when video.play is triggered and when pause() is needed) - Else I would remove `videoElement.pause() ` inside the `onVideoElementPlay ` function to have clean code and to reduce potential sources of errors in future - but only if this does not cause further errors yet.

Reply:
I'll do that :D
Good find.